src/assets/i18n/de_DE.json

```diff
@@ -2795,9 +2794,9 @@
   "account.user.new.button.create.label": "Benutzer anlegen",
   "account.user.new.firstname.error.required": "Bitte geben Sie einen Vornamen an.",
   "account.user.new.lastname.error.required": "Bitte geben Sie einen Nachnamen an.",
-  "account.user.new.login.label": "Benutzername",
-  "account.user.new.login.error.required": "Bitte geben Sie einen Benutzernamen an.",
-  "account.user.new.login.error.email": "Bitte geben Sie einen gültigen Benutzernamen an.",
+  "account.user.new.login.label": "Anmeldename",
+  "account.user.new.login.error.required": "Bitte geben Sie einen Benutzernamen an",
```

Comment:
is the wording 'Benutzername' here correct?

Reply:
No, is now fixed.